samples/javascript_typescript/02.echobot-with-counter/README.MD

```diff
@@ -5,14 +5,12 @@ This sample shows how to create a simple echo bot with state. The bot maintains
     ```bash
     git clone https://github.com/microsoft/botbuilder-samples.git
     ```
-- In a terminal, navigate to javascript_ts\2.echobot-with-counter
+- In a terminal, 
     ```bash
-    cd javascript_ts\2.echobot-with-counter
-    ```
-- Point to the MyGet feed 
-    ```bash
-    npm config set registry https://botbuilder.myget.org/F/botbuilder-v4-js-daily/npm/
+    cd samples\javascript_typescript\02.echobot-with-counter
```

Comment:
I've been using forward slashes so people on UNIX systems can still copy and paste this command in (and others)...